### DISCUSSION TITLE
My Projects Files in Qfieldcloud lost all

### DISCUSSION THREAD

**Discussion by mygiano** [opengisch/QField#3829] [Q&A]
I no longer can syncronize my project to qfieldcloud, so i check to my qfieldcloud account and showed that there's no file uploaded, but the capacity showed usage 520 MB, then there's must be files, and the changes menu still show all my changes... anybody can help??

**Comment by suricactus** [comment]
Can you please provide any information about your username and projects?

**Comment by mygiano** [comment]
username : mygiano
project name : SEPeDa
Organization : Bappelitbang-Sitaro

**Comment by Forstrevier** [comment]
Same thing with me. 2 projects, in both all files have disappeared since yesterday.

**Comment by suricactus** [comment]
@mygiano  and @Forstrevier we took your input so seriosly we paused the service until we find out what is going wrong. Please share all details what you have done before you find out the project files are missing.

**Comment by mbernasocchi** [comment]
Discussion moved to https://github.com/opengisch/QField/discussions/3838
